templates/private-net-create.yaml

```diff
@@ -30,6 +35,8 @@ resources:
       network: {get_resource: network}
       name: {get_param: private_net}
       cidr: {get_param: private_net_cidr}
+      dns_nameservers:
```

Comment:
Then this could just be dns_nameservers: {get_param: dns_nameservers} because the param itself would be a list already.